src/modules/job-ingest/job-ingest.c

```diff
@@ -103,6 +103,7 @@ struct job {
 
     char *jobspec;      // jobspec, not \0 terminated (unwrapped from signed)
     int jobspecsz;      // jobspec string length
+    json_t *jobspec_obj;
```

Comment:
i'd add a comment noting that this is not guaranteed to be a decoded representation of `job->jobspec`, b/c you've redacted environment stuff.  Could trip up someone going through code in the future.

Reply:
Thanks, agreed!  I was having the same thought.

Reply:
I called the variable `job->jobspec_redacted` in the job manager, thinking that was where maximum confusion might occur, then just noticed your comment was on the code in job-ingest.  Maybe I should add a comment there.